optuna/integration/allennlp.py

```diff
@@ -0,0 +1,86 @@
+import json
+import os
+from typing import Any
+from typing import Dict
+from typing import List
+
+import optuna
+
+try:
+    import _jsonnet
+    import allennlp.commands
+    import allennlp.common.util
+
+    _available = True
+except ImportError as e:
+    _import_error = e
+    _available = False
+    TrackerCallback = object
+
+
+class AllenNLPExecutor(object):
+    """Allennlp extension to use optuna with an allennlp config file.
+
+    Args:
+        trial:
+            A :class:`~optuna.trial.Trial` corresponding to the current evaluation
+            of the objective function.
+        config_file:
+            An allennlp config file.
+            Hyperparameters should be masked with `std.extVar`.
+            Please refer to `the config example <https://github.com/allenai/allentune/blob/
+            master/examples/classifier.jsonnet>`_.
+        serialization_dir:
+            A path which model weights and logs are saved.
+        metrics:
+            An evaluation metric for the result of ``objective``.
+
+    """
+
+    def __init__(
+        self,
+        trial: optuna.Trial,
+        config_file: str,
+        serialization_dir: str,
+        metrics: str = "best_validation_accuracy",
+        *,
+        include_package: List[str] = []
```

Comment:
Please add a description of `include_package` to the docstring.

Reply:
Added in 6ac976b.